multitenancy.md

```diff
@@ -0,0 +1,94 @@
+# Multitenancy
+
+This document contains sample instructions on how to setup one service cluster together with multiple workload clusters. As stated, the instructions below are just samples, you need update them according to your situation and requirements. You will find more details on the commands executed in the cluster and apps repos.
+
+The following repositores and commits have been tested
+- [ck8s-cluster](https://github.com/elastisys/ck8s-cluster/tree/324ac074a337b99b83a464ce0189ed2c0f267e19)
+- [compliantkubernetes-apps](https://github.com/elastisys/compliantkubernetes-apps)
```

Comment:
The link to the apps repo is pointing to `main`. In the text above it seems like you say that this should point to the specific commit that you used.

Reply:
Yes was waiting for this to be approved and merged https://github.com/elastisys/compliantkubernetes-apps/pull/145